### PR TITLE
:sparkles: Deleting all test accounts

### PR DESCRIPTION
**Pull request by mj-isip23**

### CODE REVIEW

File: TestHelpers.py [hunk 1]
```diff
@@ -17,14 +17,17 @@ def visitUrl(self, url):
 		self.setTerminalColors(NORMAL_WHITE_FONT)
 
 	def setTerminalColors(self, style):
-		print (style, end = '')
+		print(style, end = '')
```
Comment (tradrianalejandro) [inline TestHelpers.py]: planning to make this file obsolete. you can reuse helpers inside storefront folder.

BTW, please changes that folder's name from "storefront" to "stage". we'll have three folders for each environment moving forward: "stage", "pre_production", "production"

File: admin/48941_DeleteRegisteredUser.py [hunk 2]
```diff
@@ -0,0 +1,169 @@
...
+            self.printErrorMessage("Connection timeout")
+
+        time.sleep(3)
+
+    def fill_up_admin_login_form(self):
+
+        try:
+            assert self.checkIfCurrentUrlEqualToExpectedUrl(self.environment + self.locale + ADMIN_LOGIN_PAGE)
+        except AssertionError:
+            self.printErrorMessage("Current Url not equal to expected")
```
Comment (tradrianalejandro) [inline admin/48941_DeleteRegisteredUser.py]: follow console logs convention as with edit shipping details

File: admin/48941_DeleteRegisteredUser.py [hunk 3]
```diff
@@ -0,0 +1,169 @@
...
+
+        try:
+            if self.locale == "/au/":
+                self.browser.find_by_id('emailAddress').first.fill("super.admin@fitmycar.com.au")
```
Comment (tradrianalejandro) [inline admin/48941_DeleteRegisteredUser.py]: use constant variable for the email address and password.

File: admin/48941_DeleteRegisteredUser.py [hunk 4]
```diff
@@ -0,0 +1,169 @@
...
+                                        TimeoutException
+
+
+class DeleteUserAccountsFunctionTest(BrowserTester):
+
+    USER_PARTIAL_EMAIL = "testAccount"
+    locale: str
+    environment: str
+
+    def visit_locale_dashboard(self, locale, environment):
```
Comment (tradrianalejandro) [inline admin/48941_DeleteRegisteredUser.py]: use camel case for function names

File: admin/48941_DeleteRegisteredUser.py [hunk 5]
```diff
@@ -0,0 +1,169 @@
...
+
+    # delete user
+    main_class.delete_user_account()
+
+    # quit
+    time.sleep(5)
+    main_class.browser.quit()
+
+
+def main():
```
Comment (tradrianalejandro) [inline admin/48941_DeleteRegisteredUser.py]: change function name to something descriptive of the test

File: TestHelpers.py (hunk 1 above)
Comment (mj-isip23) [reply TestHelpers.py]: Done, I used Joms and Ariane's global methods and variables

File: admin/48941_DeleteRegisteredUser.py (hunk 2 above)
Comment (mj-isip23) [reply admin/48941_DeleteRegisteredUser.py]: Done

File: admin/48941_DeleteRegisteredUser.py (hunk 3 above)
Comment (mj-isip23) [reply admin/48941_DeleteRegisteredUser.py]: Done

File: admin/48941_DeleteRegisteredUser.py (hunk 4 above)
Comment (mj-isip23) [reply admin/48941_DeleteRegisteredUser.py]: Done

File: admin/48941_DeleteRegisteredUser.py (hunk 5 above)
Comment (mj-isip23) [reply admin/48941_DeleteRegisteredUser.py]: Done

File: storefront/48941_DeleteRegisteredUser.py [hunk 6]
```diff
@@ -0,0 +1,135 @@
...
+
+        if self.locale == AU_PATH:
+            self.typeInElement(self.findByID("emailAddress"), AU_ADMIN_EMAIL)
+        else:
+            self.typeInElement(self.findByID("emailAddress"), US_ADMIN_EMAIL)
+
+        self.typeInElement(self.findByID("cmCrypt"), ADMIN_PASSWORD)
+        self.findByID("btnLogin").click()
+
+        self.quit(3)
```
Comment (tradrianalejandro) [inline storefront/48941_DeleteRegisteredUser.py]: this looks like a magic number, better to declare a constant variable DEFAULT_WAITING_TIME or so

**Pull request closed without merging**